src/services_models/GroupReserve.jl

```diff
@@ -36,10 +36,7 @@ function service_requirement_constraint!(
         for r in contributing_services
     ]
 
-    # TODO: should we make a get_time_series method for the StaticReserveGroup that handles the following two lines?
-    ts_vectors =
-        [get_time_series(psi_container, s, "requirement") for s in contributing_services]
-    ts_vector = sum(hcat(ts_vectors...), dims = 2)
+    ts_vector = get_time_series(psi_container, service, "requirement")
```

Comment:
OK, this makes sense. I think you just need to add a "requirement" time series to the "init" group reserve [here](https://github.com/NREL-SIIP/PowerSimulations.jl/blob/master/test/test_services_constructor.jl#L207) to get the test to pass.

Reply:
So I wanted to make this generic for when we have a `VariableReserveGroup` just like the code for normal reserves.
However, currently, we only have `StaticReserveGroup` so there is no way to add a time-series to the PSY service.

@claytonpbarrows do you prefer I remove the time-series handling for now? I haven't figured it out how to make it so it defaults back to `ts_vector = ones(time_steps[end])` when there are no time-series added.

Reply:
Yeah, I think we should remove the time series handling since this method is specific to `StaticReserveGroup`. Then we can create another method to handle the time series for the `VariableReserveGroup`.

>I haven't figured it out how to make it so it defaults back to ts_vector = ones(time_steps[end]) when there are no time-series added.

The existing examples that default to the vector of ones are for when the entire OperationsProblem isn't using time series data. So, for now we don't support defaulting to a vector of ones when the requested time series data isn't available.

Reply:
time-series handling removed!